snippets/shorten.md

```diff
@@ -0,0 +1,23 @@
+---
+title:  shorten
+tags: string,beginner
+---
+
+Returns a shortened string
+
+Use `mb_strlen()`, `mb_substr()` and `rtrim()` to shorten a string to a give number of characters.
+
+```php
+function shorten($input, $length = 100, $end = '...')
+{
```

Comment:
Indentation should be 2 spaces per level, please fix for the whole snippet.

Reply:
Completed. (Had some issues getting .git to notice the changes but they should be there.